src/Microsoft.DotNet.Build.Tasks/CleanupVSTSAgent.cs

```diff
@@ -29,11 +29,14 @@ public class CleanupVSTSAgent : Microsoft.Build.Utilities.Task
 
         public int MaximumWorkspacesToClean { get; set; } = 8;
 
+        public bool AttemptMaxPathFolderRemoval { get; set; } = true;
```

Comment:
I don't mind changing it but I still maintain this is only going to be an attempt :)